### ISSUE TITLE
API Init container failing

### ISSUE THREAD

**Issue by BarryWilliams** [shipa-corp/helm-chart#120]
Hello,
I am getting the following error in the logs of the API init container. This is preventing the API application to start, preventing a successful install.
```shell
$ kubectl logs shipa-api-77b9c8dd4f-42fsj -c init -n shipa
Opening config file: /etc/shipa/shipa.conf
Done reading config file: /etc/shipa/shipa.conf
Warning: configuration didn't declare a database driver, using default driver.
Using "mongodb" database "shipa" from the server "shipa-mongodb-replicaset:27017".
2021-02-18 05:33:29.439857 I | Failed to process shipa license: can't accept your license, fallback to Free version
Password:
Confirm:
Error: organization already exists
```

This leaves the API containers is a crash loop:
```
$ kubectl get pods -n shipa
NAME                                    READY   STATUS                  RESTARTS   AGE
shipa-api-667df7894b-gggrs              0/1     Init:CrashLoopBackOff   5          5m14s
shipa-api-77b9c8dd4f-42fsj              0/1     Init:CrashLoopBackOff   5          5m14s
```

I have deployed with Helm following the [instructions](https://learn.shipa.io/docs/installing-shipa).

Deployed to GKE.

I saw there was a configmap that contained the ID of the organization. I edited the configmap to change the org name, then deleted the API pods, but this did not help.

I feel this log output might not be very helpful. Is there some other logs I can provide or any troubleshooting steps?

**Comment by BarryWilliams** [comment]
I did a complete redeploy - helm uninstall, delete the namespace, then start over. This worked - not that I suggest people to do this as a normal solution.

**Comment by vivek** [comment]
Hi @BarryWilliams, thanks for trying out shipa and giving your feedback.  Good to note, clean reinstall worked for you, not ideal. We have not seen in our testing. We are going to further discuss to see what might cause `Error: organization already exists` error and failed shipa-api service and see if we can reproduce and fix it. Thanks again for trying Shipa.

**Comment by vivek** [comment]
Hi @BarryWilliams, we have [released Shipa 1.2.1](https://learn.shipa.io/changelog/shipa-v121-release) and fixed this issue.  find installation instructions at https://learn.shipa.io/docs/installing-shipa and let us know if you have any questions. 

You can drop in any feedback over [slack](https://join.slack.com/t/shipa-io/shared_invite/zt-9uyk3hiu-h~FO2RI9PGOk59h3_rix6g)  and we can help you with any questions you have. Thanks for trying shipa.
